OpenRA.Mods.Common/UtilityCommands/OutputResolvedWeaponsCommand.cs

```diff
@@ -0,0 +1,46 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	public class OutputResolvedWeaponsCommand : IUtilityCommand
+	{
+		string IUtilityCommand.Name { get { return "--resolved-weapons"; } }
```

Comment:
Shouldn't this be `--resolved-weapon`?  The other two are plural because a single top-level node has multiple "rules" or "sequences", but this is outputting a singular weapons definition.